kuksa_go_client/README.md

```diff
@@ -64,4 +64,4 @@ if you encounter a problem, you have to give the protoc executable the right to
 
 ### Configuration of the KUKSA.val GO client
 For Configuration you can use the kuksa-client.json file in this directory or provide the parameters through the command line.
-If you want the GO client to work with the KUKSA.val server specify protocol as ws if you want to use the KUKSA.val databroker set protocol to grpc. You can specify the port to look for an instance of KUKSA.val either in the JSON file or through -port on the command line.
\ No newline at end of file
+If you want the GO client to work with the KUKSA.val server specify protocol as ws if you want to use the KUKSA.val databroker set protocol to grpc. On command line it's available thorugh -protocol ws/grpc.
```

Comment:
Not "thorugh" I assume? The first sentence could also benefit from rephrasing, or a dot/period after `as ws`